app/views/layouts/manage_metadata.html.erb

```diff
@@ -33,7 +33,10 @@
 
               <nav>
                 <div class="content">
-                  <h2 class="current"><%= link_to 'Manage Metadata', manage_metadata_path %></h2>
+                  <%= content_tag(:h2, class: ('current' if controller_name == 'manage_collections' || 'collection_drafts')) do %>
```

Comment:
I think we should use HTML here instead of a ruby helper for the h2

```
<h2 class="<%= 'current' if controller_name == 'manage_collections' || controller_name == 'collection_drafts' %>">
....
</h2>
```